language-server/src/dotty/tools/languageserver/worksheet/WorksheetMessages.scala

```diff
@@ -0,0 +1,27 @@
+package dotty.tools.languageserver.worksheet
+
+import java.net.URI
+
+/** The parameter for the `worksheet/exec` request. */
+case class WorksheetExecParams(uri: URI) {
```

Comment:
I suggest using org.eclipse.lsp4j.VersionedTextDocumentIdentifier which comes from the LSP instead of URI.